.travis.yml

```diff
@@ -1,9 +1,22 @@
-language: vim
+language: ruby
```

Comment:
@cirosantilli Right so it is as I imagined. I guess it is just cosmetics. I'll return it to vim if you like.